examples/ibm-is-ng/main.tf

```diff
@@ -706,3 +706,19 @@ data "ibm_is_placement_groups" "is_placement_groups" {
 ## List regions 
 data "ibm_is_regions" "regions" {
 }
+
+## Security Groups/Rules/Rule
+// Create is_security_groups data source
+data "ibm_is_security_groups" "is_security_groups_instance" {
+}
+
+// Create is_security_group_rule data source
+data "ibm_is_security_group_rule" "is_security_group_rule_instance" {
+  security_group_id = var.is_security_group_rule_security_group_id
```

Comment:
fix examples according to updated schema

Reply:
done.